packages/polyfills-loader/src/types.d.ts

```diff
@@ -56,6 +58,8 @@ export interface PolyfillConfig {
   minify?: boolean;
   // code used to initialze the module
   initializer?: string;
+  // order of the polyfill when loaded, defaults to 50
+  order?: number;
```

Comment:
why would you use an order number in the object and not relay on the native array order?
🤔

Reply:
Good question!  

The input of the function is unfortunately an object, not an array. This means that currently, when users define the polyfills, it doesn't matter what order they use when defining the properties. My first idea was indeed to use the order of the properties within the object, but there were some drawbacks:
- it would introduce a breaking change, as order of the polyfills would change
- some properties on the object are not polyfills, but additional options (`hash`, `custom`)
- custom polyfills would still be together, ie. you cannot put them and before and after. (currently custom polyfills are only allowed to be before all others)

Hope this makes sense 😊

Reply:
I don't think we should expose this as a public API. I think for the predefined polyfills, we should sort them in the order that we know about. For the custom polyfills we can keep the array order.

Reply:
> I don't think we should expose this as a public API. I think for the predefined polyfills, we should sort them in the order that we know about. We can keep an array with the order for that. For the custom polyfills we can keep the array order.

This doesn't solve the use case of having custom polyfills after the predefined ones. How should we solve this?

Reply:
I'd load custom polyfills after the predefined ones. Do we have any use cases for other behavior?

Reply:
To be honest, not sure. 🤔   
Currently, we load them before the predefined ones, and as pushing these forward is a breaking change, this might break apps of other users. I could imagine people using these custom polyfills to configure some of the polyfills (eg. webcomponents, by adding flags to its global objects)

So, I see a couple of solutions:
- load custom polyfills afterwards, but don't consider this as a breaking change
- load custom polyfills afterwards, and make it as a breaking change
- create some logic to support both before and after

PS: if we remove the order functionality altogether, I don't even need an internal array for the predefined ones, as we add these polyfills already in a predefined way.

Reply:
In the end, I'll change it to the following: remove order and load custom polyfills afterwards. As all polyfills are loaded in parallel in the loader project, it will not break anything for current users 😊